packages/suite/src/views/wallet/send/components/Options/BitcoinOptions/BitcoinOptions.tsx

```diff
@@ -115,33 +114,6 @@ export const BitcoinOptions = () => {
                             </StyledButton>
                         </Tooltip>
                     )}
-
-                    {isFeatureFlagEnabled('RBF') &&
-                        network.features?.includes('rbf') &&
-                        !locktimeEnabled && (
-                            <Tooltip
-                                addon={
-                                    <OpenGuideFromTooltip id="/3_send-and-receive/5_replace-by-fee-rbf.md" />
```

Comment:
Reminder: remove the article from Guide

Reply:
[Knowledge Base](https://trezor.io/learn/a/replace-by-fee-rbf-bitcoin) must also be updated, I notified @b-irving.